src/ecpair.js

```diff
@@ -66,11 +66,12 @@ ECPair.fromWIF = function (string, network) {
 
     network = network.filter(function (network) {
       return version === network.wif
-    }).pop() || {}
+    }).pop()
+    if (!network) throw new Error('Invalid network version')
```

Comment:
Will re-check this in the next 24 hours,  suspect its all OK just want to double check.